### DISCUSSION TITLE
Two small questions about NavigationView

### DISCUSSION THREAD

**Discussion by Dawson924** [amwx/FluentAvalonia#540] [Q&A]
Does it have built-in animation when content changed? (_just like the frame_)

How can I disable the **tooltips** that comes up while the pointer is on a navigationview item? Because I found out that if I change RequestedThemeVariant, the theme of the tooltip will not follow the changes

**Comment by Dawson924** [comment]
One of the solution that I m doing right now is to set ShowDelay=a large number.

**Comment by timunie** [comment]
> How can I disable the tooltips that comes up while the pointer is on a navigationview item? Because I found out that if I change RequestedThemeVariant, the theme of the tooltip will not follow the changes

I believe this is an upstream issue where no real solution was found (yet)
